drivers/hmis/app/models/hmis/unit.rb

```diff
@@ -22,6 +24,10 @@ class Hmis::Unit < Hmis::HmisBase
   has_many :unit_occupancies, class_name: 'Hmis::UnitOccupancy', inverse_of: :unit, dependent: :destroy
   has_many :active_unit_occupancies, -> { active }, class_name: 'Hmis::UnitOccupancy', inverse_of: :unit
   has_many :current_occupants, through: :active_unit_occupancies, class_name: 'Hmis::Hud::Enrollment', source: :enrollment
+  has_many :opportunities, as: :owner, class_name: 'Hmis::Ce::Opportunity', inverse_of: :owner
```

Comment:
I think we need dependent: :destroy here. Calling `unit.destroy!` should destroy any associated opportunities.